src/core/sent_packet_metadata.h

```diff
@@ -80,10 +80,6 @@ typedef struct QUIC_SENT_FRAME_METADATA {
 
 } QUIC_SENT_FRAME_METADATA;
 
-CXPLAT_STATIC_ASSERT(
```

Comment:
Why is this moved away from the frame metadata type, which is what the comment is referring to?

Reply:
I had to move it because of header include order changes. It would be _really nasty_ to try to keep it here.